### ISSUE TITLE
Mixing USER and USER_INT include assets twice. 

### ISSUE THREAD

**Issue by sabbelasichon** [sabbelasichon/typo3_encore#142]
Have a look at the asset collector or leverage the possibilities of the asset collector

**Comment by Gitsko** [comment]
Does a workaround exist here by any chance?

**Comment by sabbelasichon** [comment]
Not yet. Do you have any suggestions here?

**Comment by Gitsko** [comment]
Not really, I see the only way to include the assets at the moment is via the <f:asset> viewhelper, but with that I can't use hashed filenames, the solution will be really the asset collector in the extension

**Comment by akiessling** [comment]
I can confirm the behavior for 12.4.19 as well. 
Moving the JS inclusion from TS to the Fluid Layout seems to have fixed it in my project, so the TSFE setup won't run twice
